### ISSUE TITLE
Let vs-size-property fallback to vs-repeat height

### ISSUE THREAD

**Issue by diabox** [kamilkp/angular-vs-repeat#38]
First of all, thanks for this wonderful directive!
It would be great if it could use the size defined in `vs-repeat` if the current item does not contain the `vs-size-property`.
```
$scope.sizes = originalCollection.map(function(item){
   return item[$attrs.vsSizeProperty] || $scope.elementSize; //# Line 165
});
```

**Comment by kamilkp** [comment]
This is done differently now. Check out the newest version

**Comment by kamilkp** [comment]
closed via 7445e1e
